website/versioned_docs/version-3.0/examples/export_entire_dataset.ts

```diff
@@ -0,0 +1,3 @@
+import { Dataset } from 'crawlee';
+
+await Dataset.exportToValue('OUTPUT', { contentType: 'text/csv', keyValueStoreName: 'my-data' });
```

Comment:
i'd make the example a bit more verbose
- first create some dataset items
- then export
- add comments to each stage
- add the missing \n at the end of file :P